eoscostarica/eos-rate
# Create ci cd production

### GH Issue

# Adapts the kubernetes deployment for a production environment. refs #404 

* removes hasura from the build script, there is actually no need to build it manually.
* adds hasura secrets to store graphql password.
* adds secrets for hasura deployment.
* adds a production build for the frontend.
* adds ignore .env.production.
* creates and deploys a new production namespace.

### Steps to test

1. execute scripts/deploy.sh production ${public_ip}
1. visit eosrate.io
1. you should see the eos rate page

### Checklist
